eipw-lint/src/lib.rs

```diff
@@ -1002,6 +1002,12 @@ fn process<'a>(
         } else {
             data.sourcepos.start.line += preamble_lines;
         }
+
+        if data.sourcepos.end.line == 0 {
+            data.sourcepos.end.line = data.sourcepos.start.line;
```

Comment:
Note to self: if `end.column` is non-zero and less than `start.column`, this line causes problems (and I've seen it happen.)